delfin/tests/unit/api/fakes.py

```diff
@@ -329,7 +329,7 @@ def fake_access_infos_show_all(context):
     return [access_info]
 
 
-def fake_update_access_info(self, context, access_info):
+def fake_update_access_info(self, context):
```

Comment:
You need to update caller of this function too. (Please check, file delfin/tests/unit/api/v1/test_access_info.py)

Reply:
done